egs/wsj/s5/steps/libs/nnet3/train/common.py

```diff
@@ -502,13 +502,34 @@ def smooth_presoftmax_prior_scale_vector(pdf_counts,
 
 
 def prepare_initial_network(dir, run_opts, srand=-3):
-    if os.path.exists(dir+"/configs/init.config"):
+    if not os.path.exists("{0}/configs/init.config".format(dir)):
```

Comment:
We still check existence of init.config and use it for stage -4 (init.raw generation) and stage -2 (lda generation)